test/support/browser_helper.rb

```diff
@@ -28,6 +35,20 @@ class BrowserTestCase < Minitest::Spec
   end
 
   def setup
+    debug = ENV['DEBUG'] == '1'
+    driver_options = {
+      js_errors: false,
+      logger: NilLogger.new,
+      phantomjs_logger: STDERR,
+      debug: false,
+      phantomjs_options: ['--load-images=no', '--ignore-ssl-errors=yes', '--web-security=false'],
```

Comment:
the `--web-security` bit allows us to fire XHR requests directly in JS to work around a wonky interaction with clicking the confirmation `Okay` button.